src/components/Photo/DocumentAutoCapture.js

```diff
@@ -9,6 +9,7 @@ import { DocumentOverlay } from '../Overlay'
 import Camera from '../Camera'
 import CameraError from '../CameraError'
 import { postToBackend } from '~utils/sdkBackend'
+import style from '../Camera/style.scss'
```

Comment:
So I still think that `.docAutoCaptureFrame` should live locally next to this component in a separate `style.scss` 😏

Reply:
My initial approach was to make `.docAutoCaptureFrame` part of `src/components/Photo/style.scss` (I think that's what you mean?) but that didn't work since the selector I had in `src/components/Camera/style.scss` didn't recognise `.docAutoCaptureFrame` at that level due to the generated CSS classes. But let's chat if that's not what you meant. 🤔

Reply:
I mean this:

```
// src/components/Photo/style.scss

.docAutoCaptureFrame
  display: block;
}
```

```
// src/components/Photo/DocumentAutoCapture.js

import style from './style.scss'

...
<Camera className={style.docAutoCaptureFrame} />
...
```

But I'm OK with this approach too.